full-service/src/service/balance.rs

```diff
@@ -215,34 +216,47 @@ where
     }
 
     fn get_network_status(&self) -> Result<NetworkStatus, BalanceServiceError> {
+        let (network_block_height, fees, block_version) = match self.offline {
+            true => {
+                let mut fees = BTreeMap::new();
+                fees.insert(Mob::ID, Mob::MINIMUM_FEE);
+                fees.insert(TokenId::from(1), 1024);
```

Comment:
Where does this 1024 come from? 

FWIW [in Moby the eUSD fee is $0.00256, or 2560 micro-eUSD tokens](https://github.com/mobilecoinofficial/mobilecoin-app/blob/4cd4611e6320eeedc6d938cbab87b12e95245cd5/lib/domain/mobilecoin_config.dart#L88)

Reply:
Thanks for catching that, it's 1024 by default. Will update to reflect that. Is there anywhere this is defined in the mc library?

Reply:
Not that I know of.